regression/contracts/frees-clause-while-loop-fail/test.desc

```diff
@@ -0,0 +1,10 @@
+CORE
+main.c
+--apply-loop-contracts
+^main.c.* error: frees clause target must be a pointer-typed expression or a call to a function returning void$
```

Comment:
Does the error message says the name and the type of the parameter an user tried to use with `__CPROVER_frees`? It might be really helpful for debugging.